examples/all-clusters-app/esp32/sdkconfig.defaults

```diff
@@ -52,4 +52,7 @@ CONFIG_LWIP_HOOK_ND6_GET_GW_DEFAULT=y
 
 # Serial Flasher config
 CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
-CONFIG_ESPTOOLPY_FLASHSIZE="4MB"
\ No newline at end of file
+CONFIG_ESPTOOLPY_FLASHSIZE="4MB"
+
+# platform mdns
+CONFIG_USE_MINIMAL_MDNS=n
```

Comment:
@wqx6 Why have we set this in all example sdkconfigs rather than setting its default value to y in Kconfig?

Reply:
Done